prepare_setup.sh

```diff
@@ -299,6 +312,9 @@ main(){
     packages_install
     let status=$status+$?
 
+    pip_packages_install
```

Comment:
Do we need to continue installation if `package_install` step failed?

Reply:
The script behaviour is to continue when even if there is a failure, it would be good to refactor it in another PR